cmd/minimega/disk.go

```diff
@@ -328,6 +355,39 @@ func parseInjectPairs(files []string) (map[string]string, error) {
 	return pairs, nil
 }
 
+// parseFiles parses the files argument passed into the 'disk inject ...'
+// command. if options are used, the files argument gets turned into a
+// minicli.Command StringArg, otherwise it gets turned into a ListArg.
+// this function returns either a paths string slice if 'delete' is part
+// of the original command, or a pairs string map.
+func parseFiles(files interface{}) (map[string]string, []string, error) {
+	var pairs map[string]string
+	var paths []string
+	var err error
+	switch v := files.(type) {
+	case []string:
+		if !(sliceContainsCharacters(v, ",:")) {
```

Comment:
Final final thing I promise :) 

With this change we no longer get the error "malformed command; expected src:dst pairs" when running `disk inject /scratch/images/test.qcow2 files /bin/foo`.

Maybe the code should just be explicit whether it expects single files or pairs based on the delete flag?

Reply:
@jacdavi I changed the logic a bit to handle this case and tried all the previous ones you had asked about. I think it covers them all. Let me know what you think.

Reply:
@jacdavi bump

Reply:
Thanks for the reminder, looks good now!